### PR TITLE
Adds what's new 

### PR DESCRIPTION
**Pull request by lysannep**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/error-reporting/whats-new.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+In addition to filtering attributes by name, you can now use the global filter bar to filter attributes by description.
+
+## Sorting Users (2022-07-29)
+In the Organization settings, you can now sort the Users page by username, email, and role.
+
+## Releases Comparison View (2021-10-05)
+The Releases Comparison View allows you to easily monitor and compare stability, user adoption, and other crash and error metrics across all versions of your application. 
+
+## Project Overview Page (2021-06-04)
+Quickly monitor your project’s health with the Backtrace Web Console’s newest view:  The Project Overview.  See total errors count, errors through time, errors by app version, errors by platform, and much more all on one page.
```
Comment (aoleszyn) [inline docs/error-reporting/whats-new.md:22]:
```suggestion
Quickly monitor your project’s health with the Backtrace Web Console’s newest view:  The Project Overview. See total errors count, errors through time, errors by app version, errors by platform, and much more all on one page.
```

File: docs/error-reporting/whats-new.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## Attachment Preview (2021-04-30)
+If you add screenshots, logs, or other attachments to your errors, you can now quickly view them the Debugger
```
Comment (aoleszyn) [inline docs/error-reporting/whats-new.md:25]:
```suggestion
If you add screenshots, logs, or other attachments to your errors, you can quickly view them on the Debugger.
```

File: docs/error-reporting/whats-new.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+## Submission Actions (2021-03-16)
+As an admin user, you may be interested in "Submission Actions". You'll find them under Project Settings / Submission Actions. Submission actions can configure projects to: Automate assignment, Set tags and attributes, Reject submission traffic, and more!
```
Comment (aoleszyn) [inline docs/error-reporting/whats-new.md:28]:
```suggestion
As an admin user, you may be interested in "Submission Actions". You'll find them under Project Settings / Submission Actions. Submission Actions can configure projects to: Automate assignments, Set tags and attributes, Reject submission traffic, and more!
```

File: docs/error-reporting/whats-new.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+## Game Console Connectors (2021-01-10)
+Backtrace now has exposes additional support to manage crashes and errors for PlayStation4, PlayStation5, Microsoft Xbox One, Series X|S, and Nintendo Switch.
```
Comment (aoleszyn) [inline docs/error-reporting/whats-new.md:31]:
```suggestion
Backtrace now exposes additional support to manage crashes and errors for PlayStation4, PlayStation5, Microsoft Xbox One, Series X|S, and Nintendo Switch. 
```

**Review by aoleszyn — CHANGES_REQUESTED** [review]
Couple minor changes.

**Review by aoleszyn — APPROVED** [review]

**Review by spider-sauce — APPROVED** [review]

**Pull request merged**